README.md

```diff
@@ -1,61 +1,57 @@
-# atlas-action
+# GitHub Actions for Atlas
 
-A GitHub Action for [Atlas](https://github.com/ariga/atlas).
+This repository contains GitHub Actions for working with [Atlas](https://atlasgo.io).
 
-This action is used for [linting migration directories](https://atlasgo.io/versioned/lint)
-using the `atlas migrate lint` command. This command  validates and analyzes the contents
-of migration directories and generates insights and diagnostics on the selected changes:
+If you are looking for the old TypeScript-based action, please see the old [README](doc/typescript-action.md). 
 
-* Ensure the migration history can be replayed from any point at time.
-* Protect from unexpected history changes when concurrent migrations are written to the migration directory by
-  multiple team members.
-* Detect whether destructive or irreversible changes have been made or whether they are dependent on tables'
-  contents and can cause a migration failure.
+To learn more about the recommended way to build workflows, read our guide on 
+[Modern CI/CD for Databases](https://atlasgo.io/guides/modern-database-ci-cd).
 
-### Supported directory formats
+## Actions
 
-This action supports analyzing migration directories in formats
-accepted by different schema migration tools:
-* [Atlas](https://atlasgo.io)
-* [golang-migrate](https://github.com/golang-migrate/migrate)
-* [goose](https://github.com/pressly/goose)
-* [dbmate](https://github.com/amacneil/dbmate)
-* [Flyway](https://flywaydb.org/)
-* [Liquibase](https://www.liquibase.org/)
+| Action                                             | Description                                             |
+|----------------------------------------------------|---------------------------------------------------------|
+| [ariga/setup-atlas](#setup-atlas)                  | Setup the Atlas CLI and optionally login to Atlas Cloud |
```

Comment:
should we move it to this repo too?

Reply:
Not sure, I think it looks nice actually :-)